runtime/metricsview/executor_rewrite_approx_comparisons.go

```diff
@@ -46,6 +46,14 @@ func (e *Executor) rewriteApproxComparisonNode(a *AST, n *SelectNode) bool {
 	}
 	sortField := a.Root.OrderBy[0]
 
+	// if dim does not have explicit unnest, then don't optimize
+	// for example druid fails with join on cte having multi value dimension
+	for _, dim := range n.FromSelect.DimFields {
+		if dim.AutoUnnest {
+			return false
+		}
+	}
```

Comment:
Could we just check `if len(a.unnests) > 0 { return false }` and then avoid the `AutoUnnest` field on `FieldNode`?

Reply:
ok removed, I wasn't sure if dim fields on parent select node will always be same as dim fields in FromSelect.